docs/architecture/adr-016-validator-consensus-key-rotation.md

```diff
@@ -0,0 +1,120 @@
+# ADR 016: Validator Consensus Key Rotation
+
+## Changelog
+
+- 2019 Oct 23: Initial draft
+- 2019 Nov 28: Add key rotation fee
+
+## Context
+
+Validator consensus key rotation feature has been discussed and requested for a long time, for the sake of safer validator 
+key management policy (e.g. https://github.com/tendermint/tendermint/issues/1136). So, we suggest one of the simplest form of
+validator consensus key rotation implementation mostly onto Cosmos-SDK.
```

Comment:
thanks for noting that. I add below sentences to clarify the reason.

_ We don't need to make any update on consensus logic in Tendermint because Tendermint does not have any mapping information of consensus key and validator operator key, meaning that from Tendermint point of view, a consensus key rotation of a validator is simply a replacement of a consensus key to another._